src/commands/pickFuncProcess.ts

```diff
@@ -3,32 +3,58 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// tslint:disable-next-line:no-require-imports
-import ps = require('ps-node');
-import { isNullOrUndefined } from 'util';
 import * as vscode from 'vscode';
+import { Task, TaskExecution } from 'vscode';
 import { IActionContext, UserCancelledError } from 'vscode-azureextensionui';
 import { extensionPrefix, isWindows } from '../constants';
 import { validateFuncCoreToolsInstalled } from '../funcCoreTools/validateFuncCoreToolsInstalled';
 import { localize } from '../localize';
 import { getFuncExtensionSetting } from '../ProjectSettings';
-import { cpUtils } from '../utils/cpUtils';
-import { funcHostTaskId } from './createNewProject/IProjectCreator';
+import { tryFetchNodeModule } from '../utils/tryFetchNodeModule';
+import { funcHostTaskLabel } from './createNewProject/IProjectCreator';
 
 export async function pickFuncProcess(actionContext: IActionContext): Promise<string | undefined> {
     if (!await validateFuncCoreToolsInstalled(true /* forcePrompt */)) {
         throw new UserCancelledError();
     }
 
-    let funcHostPid: string | undefined = await getFuncHostPid();
-    if (funcHostPid !== undefined) {
-        // Stop the functions host to prevent build errors like "Cannot access the file '...' because it is being used by another process."
-        await killProcess(funcHostPid);
+    // Stop any running func task so that a build can access those dlls
+    await stopFuncTaskIfRunning();
+
+    const tasks: Task[] = await vscode.tasks.fetchTasks();
+    const funcTask: Task | undefined = tasks.find(isFuncTask);
+    if (!funcTask) {
+        throw new Error(localize('noFuncTask', 'Failed to find task with label "{0}".', funcHostTaskLabel));
```

Comment:
name?

Reply:
The property defined in tasks.json is "label" not "name", so I'll leave as-is to match that